truss-chains/truss_chains/public_types.py

```diff
@@ -363,6 +363,13 @@ def get_spec(self) -> AssetSpec:
         return self._spec.model_copy(deep=True)
 
 
+try:
```

Comment:
Good catch! I'm assuming this was the root cause to https://basetenlabs.slack.com/archives/C04NM3YA2DT/p1742431668200519?